### PR TITLE
Implement profiling page for Panel

### PR DESCRIPTION
**Pull request by philippjfr**
Adds an optional `/admin` endpoint to Panel with profiling output. This will expand to a number of visualization and views to monitor a live Panel server, including seeing currently active sessions, logging output, callbacks, and much more.

Screen Shot 2021-08-18 at 7 59 23 PM

### CODE REVIEW

**Comment by MarcSkovMadsen** [comment]
Do I see the terminal in action?

**Comment by MarcSkovMadsen** [comment]
For inspiration check also scalene out @philippjfr . https://github.com/plasma-umass/scalene

**Comment by philippjfr** [comment]
@MarcSkovMadsen scalene indeed looks great but I see no way to actually run it from within an existing Python session. The only way I found to invoke it required forking the process and running a code snippet in the forked process.

File: doc/user_guide/index.rst [hunk 1]
```diff
@@ -65,6 +65,10 @@ when needed.
 
 `Authentication <Authentication.html>`_
  Learn how to add an authentication component in front of your application.
+
+`Profiling and Debugging <Profiling_and_Debugging.html>`_
```
Comment (philippjfr) [inline doc/user_guide/index.rst:69]:
```suggestion
`Profiling and Debugging <Performance_and_Debugging.html>`_
```

**Comment by MarcSkovMadsen** [comment]
For me the main take away is also the look and frem which looks great.

**Pull request merged**